static/scripts/rewards/web3/erc20-permit.ts

```diff
@@ -100,12 +100,17 @@ async function transferFromPermit(permit2Contract: Contract, app: AppState) {
 async function waitForTransaction(tx: TransactionResponse) {
   try {
     const receipt = await tx.wait();
+    viewClaimButton.onclick = () => {
+      window.open(`https://blockscan.com/tx/${receipt.transactionHash}`, "_blank");
```

Comment:
I would prefer to use etherscan, or gnosisscan 

However, are you adding this because it supports all the main networks? If so then it might be the best solution.

Reply:
Yes exactly it's the most robust solution that instantly supports our mainnets. The page will open up with a large CTA taking them to the underlying explorer so gnosisscan or etherscan for example but will display all the tx details too